htdocs/core/lib/functions.lib.php

```diff
@@ -1658,8 +1658,10 @@ function dol_syslog($message, $level = LOG_INFO, $ident = 0, $suffixinfilename =
 	if (!empty($message)) {
 		// Test log level
 		$logLevels = array(LOG_EMERG=>'EMERG', LOG_ALERT=>'ALERT', LOG_CRIT=>'CRITICAL', LOG_ERR=>'ERR', LOG_WARNING=>'WARN', LOG_NOTICE=>'NOTICE', LOG_INFO=>'INFO', LOG_DEBUG=>'DEBUG');
+
 		if (!array_key_exists($level, $logLevels)) {
-			throw new Exception('Incorrect log level');
+			dol_syslog($langs->trans('ErrorBadLogLevel', $level), LOG_ERR);
+			$level = getDolGlobalInt('DEFAULT_SYSLOG_LEVEL', $logLevels[LOG_ERR]);
```

Comment:
Why not just doing $level = 'ERR';
We must avoid the use fo getDolGlobalInt because this method may not be defined or env may not be correctly loaded to have it working.
Also if the log level is wrong, it means there is acritical bug in code so we can directly report en error with level ERR.